src/test/java/org/elasticsearch/transport/netty/KeyedLockTests.java

```diff
@@ -75,6 +77,58 @@ public void checkIfMapEmptyAfterLotsOfAcquireAndReleases() throws InterruptedExc
         }
     }
 
+    public void checkIfFairLockingWorks() throws InterruptedException {
```

Comment:
not sure what this test tests  to be honest but we only pass the param on so no need to really test it. the randomization up there is fine.